### PR TITLE
[4.0] Update component manifests

### PR DESCRIPTION
**Pull request by brianteeman**
Based on the work by @laoneo in #21817

Code review only

### CODE REVIEW

File: administrator/components/com_admin/admin.xml [hunk 1]
```diff
@@ -1,23 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<extension type="component" version="3.1" method="upgrade">
+<extension type="component" version="4.0" method="upgrade">
 	<name>com_admin</name>
 	<author>Joomla! Project</author>
 	<creationDate>April 2006</creationDate>
```
Comment (laoneo) [inline administrator/components/com_admin/admin.xml:5]: Wondering if the creation date should also being changed?

Comment (brianteeman) [reply administrator/components/com_admin/admin.xml:5]: i just went by your com_content manifest - it is above my paygrade

Comment (wilsonge) [reply administrator/components/com_admin/admin.xml:5]: Creation date hasn't changed

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks - i guess i need to check modules and plugins now

**Pull request merged**